### PR TITLE
[RAPTOR-7539]Add built-in support for ONNX

### PR DESCRIPTION
**Pull request by aslisabanci**
## Summary
Add built-in support for ONNX models.

## Rationale
To provide 1st class support for models with .onnx extension. This will allow our users to run their ONNX models on DataRobot with minimal effort, using the Python3 ONNX drop-in environment that contains the `onnxruntime`.

### CODE REVIEW

**Comment by aslisabanci** [comment]
jarvis pls test cmrunner_integration

**Comment by aslisabanci** [comment]
jarvis pls run functional_drop_in

**Comment by aslisabanci** [comment]
jarvis pls run functional_inference_model_templates

**Comment by aslisabanci** [comment]
jarvis pls run functional_inference_model_templates

File: tests/functional/test_inference_model_templates.py [hunk 1]
```diff
@@ -107,6 +107,17 @@ class TestInferenceModelTemplates(object):
                 None,
                 None,
             ),
+            (
+                "python3_onnx",
```
Comment (yakov-g) [inline tests/functional/test_inference_model_templates.py:111]: looks like this should be model path. e.g. python3_onnx_regression

**Comment by aslisabanci** [comment]
jarvis pls run functional_inference_model_templates

**Comment by aslisabanci** [comment]
jarvis pls run functional_inference_model_templates

**Comment by aslisabanci** [comment]
jarvis pls run functional_drop_in, cmrunner_integration, functional_training_model_templates

**Comment by jmbannon** [comment]
Can you add all the ONNX-specific files to be owned by custom models in the DRCODEOWNERS file:
https://github.com/datarobot/datarobot-user-models/blob/master/DRCODEOWNERS

**Review by RodrigoGonzalez — APPROVED** [review]
LGTM CORE MODELING

**Pull request merged**